README.md

```diff
@@ -7,29 +7,37 @@ See video: https://www.youtube.com/watch?v=MbYySNSVVFo
 
 Capture sound waves with microphone of computer and then represent this wave on leds of arduino using Pingo.io.
 
-## How i install/use this project?
+## What do you need?
+The hardware used is an Arduino Uno and the components are:
+- 6 120 Ω resistors;
+- 6 3mm LEDs;
+- Some jumpers;
+- Some noise.
 
+## How i install/use this project?
 When you use pingo/pyfirmata we need preload StandardFirmata on your board, on ARDUINO IDE preload this example on menus:
 
 ```"File" -> "Examples" -> "Firmata" -> "StandartFirmata"```
 
-We need connect cables and leds on your Arduino :
 
-![Arduino](/contrib/draw.png "Arduino")
+## Arduino Setup
 
-Clone this project:
+![Arduino](https://raw.githubusercontent.com/akarokr/pyputsputs/master/contrib/draw.png "Arduino")
+
+## Clone the project
 
 ```
 $ git clone git@github.com:rafaelhenrique/pingo_experiments.git
 ```
 
-Create an virtualenv (i prefer [virtualenv-wrapper](https://virtualenvwrapper.readthedocs.org/en/latest/ "virtualenv-wrapper")):
+Create an virtualenv (I like the [virtualenv-wrapper](https://virtualenvwrapper.readthedocs.org/en/latest/ "virtualenv-wrapper")):
 
 ```
-$ mkvirtualenv pyputsputs
+$ mkvirtualenv -p /usr/bin/python2.7 pyputsputs
```

Comment:
The initial description needs be changed not the command, generally default version of python is a version 2.X on all linux distros. If you specify this version maybe on other operational systems this command not works fine.